src/components/KCard/KCard.vue

```diff
@@ -1,244 +1,122 @@
 <template>
-  <section
-    :aria-describedby="contentId || undefined"
-    :aria-label="title ? title : undefined"
-    :aria-labelledby="!title && slots.title ? titleId : undefined"
-    class="kong-card"
-    :class="[borderVariant, {'hover': hasHover, 'kcard-shadow': hasShadow }]"
-  >
+  <section class="k-card">
     <div
-      v-if="showCardHead"
-      class="k-card-header"
-      :class="{ 'has-status': status || slots.statusHat }"
+      v-if="$slots.title || title || $slots.actions"
+      class="card-header"
     >
       <div
-        v-if="status || slots.statusHat"
-        class="k-card-status-hat"
-      >
-        <!-- @slot Use this slot to pass status text above title -->
-        <slot name="statusHat">
-          {{ status }}
-        </slot>
-      </div>
-
-      <div
-        v-if="showCardTitleWithoutStatus"
-        :id="title ? undefined : titleId"
-        class="k-card-title"
+        v-if="$slots.title || title"
+        class="card-title"
       >
         <h4>
```

Comment:
question: why do we use a header tag?

Reply:
I guess that's what I would use for a card title. Removed it so make it just a styled div